### PR TITLE
Events tracking

### LINKED ISSUE #486: Start tracking events in Plausible Analytics
Dependant on: https://github.com/oceanprotocol/team-sailfish/issues/21

Since we decided that we can move forward with Plausible, we should set up the events in df-web. 

Steps:

- [ ] Decide which events to track
- [ ] Decide the user journeys that we want to follow
- [ ] Add the tracking tags within the frontend 
- [ ] All basic end-to-end funnels have been setup and delivered

### PR DESCRIPTION
**Pull request by jamiehewitt15**
Fixes #486 

Changes proposed in this PR:

- Updating tracking script so that it includes the event tracking
- Adding classnames to every button to that they can potentially be tracked from the UI

### CODE REVIEW

File: src/components/common/ClaimItem.svelte [hunk 1]
```diff
@@ -39,13 +39,15 @@
         text={redirectLink.text}
         onclick={() => navigate(redirectLink.url)}
         disabled={disableRedirect}
+        class="plausible-event-name=Button+Redirect+link"
```
Comment (KatunaNorbert) [inline src/components/common/ClaimItem.svelte:42]: can we be more specific here? Ex: `Button+Redirect+to+veOCEAN+link`

File: src/components/common/Row.svelte [hunk 2]
```diff
@@ -35,6 +35,7 @@
             onclick={() =>
               clickData ? button.onClick(clickData) : button.onClick(rowObject)}
             disabled={button.disabled}
+            class="plausible-event-name=Button+row+object+click"
```
Comment (KatunaNorbert) [inline src/components/common/Row.svelte:38]: I wouldn't add an event here, this is not specific enough

File: src/components/common/WalletConnectModal.svelte [hunk 3]
```diff
@@ -26,6 +26,7 @@
             );
           }}
           disabled={loading}
+          class="plausible-event-name=Button+Close+Button"
```
Comment (KatunaNorbert) [inline src/components/common/WalletConnectModal.svelte:29]: let's make this more specific, ex: `Button+Close+WalletConnect+Modal`

File: src/components/data/DataRow.svelte [hunk 4]
```diff
@@ -36,7 +36,8 @@
           <Button
             text={button.text}
             onclick={() => button.onClick(pool)}
-            disabled={button.disabled}
+            disabled={button.disabled} 
+            class="plausible-event-name=Button+row+data"
```
Comment (KatunaNorbert) [inline src/components/data/DataRow.svelte:40]: we can remove this one, not specific enough

**Comment by KatunaNorbert** [comment]
Other useful events:
- on links
- on table header(for sorting)
- table dropdowns( column, table sorting)

File: src/components/data/DataRow.svelte (hunk 4 above)
Comment (jamiehewitt15) [reply src/components/data/DataRow.svelte]: done

File: src/components/common/WalletConnectModal.svelte (hunk 3 above)
Comment (jamiehewitt15) [reply src/components/common/WalletConnectModal.svelte]: done

File: src/components/common/Row.svelte (hunk 2 above)
Comment (jamiehewitt15) [reply src/components/common/Row.svelte]: removed

**Review by KatunaNorbert — APPROVED** [review]
Looks good, let's see it in action!

**Pull request merged**